File: benchmark/2026-09-17-7677581(latest)/llm-intrinsic/trials/I1/typescript/trial_01/candidate.ts
function collect(count: number): number[] {
    const out: number[] = [];
    let state: number = 7;
    for (let step: number = 0; step < count; step = step + 1) {
        state = (state * 48271) % 2147483647;
        out.push(state % 1000);
    }
    return out;
}

function bigger(left: number, right: number): number {
    if (left > right) {
        return left;
    } else {
        return right;
    }
}

function main(): void {
    const items: number[] = collect(50);
    let total: number = 0;
    let biggest: number = items[0];
    let evens: number = 0;
    for (const item of items) {
        total = total + item;
        biggest = bigger(biggest, item);
        if (item % 2 === 0) {
            evens = evens + 1;
        }
    }
    const pieces: string[] = [];
    for (let index: number = 0; index < 5; index = index + 1) {
        pieces.push(String(items[index]));
    }
    const joined: string = pieces.join("-");
    console.log("SUM " + String(total));
    console.log("MAX " + String(biggest));
    console.log("EVENS " + String(evens));
    console.log("JOINED " + joined);
}

main();

File: benchmark/2026-09-17-7677581(latest)/llm-intrinsic/trials/I1/python/trial_01/candidate.py
def collect(count: int) -> list[int]:
    out: list[int] = []
    state: int = 7
    for step in range(count):
        state = (state * 48271) % 2147483647
        out.append(state % 1000)
    return out


def bigger(left: int, right: int) -> int:
    if left > right:
        return left
    else:
        return right


def main() -> None:
    items: list[int] = collect(50)
    total: int = 0
    biggest: int = items[0]
    evens: int = 0
    for item in items:
        total = total + item
        biggest = bigger(biggest, item)
        if item % 2 == 0:
            evens = evens + 1
    pieces: list[str] = []
    for index in range(5):
        pieces.append(str(items[index]))
    label: str = "-".join(pieces)
    print("SUM " + str(total))
    print("MAX " + str(biggest))
    print("EVENS " + str(evens))
    print("JOINED " + label)


main()
